.github/workflows/build-windows.yml

```diff
@@ -0,0 +1,297 @@
+          # }
+          # if ($env:TILEDB_CI_TSAN -eq "ON") {
+          #  $bootstrapOptions = $bootstrapOptions + " -EnableSanitizer thread -EnableDebug"
+          # }
+          # static already added above as initial default
+          # if ($env:TILEDB_FORCE_BUILD_DEPS" -eq "ON") {
+          #   $bootstrapOptions = $bootstrapOptions + " -EnableBuildDeps"
+          # }
+          if ($env:TILEDB_WERROR -eq "OFF") {
+            $bootstrapOptions = $bootstrapOptions + " -DisableWerror"
+          }
+          
+          $bootstrapExpression = "& $env:BUILD_SOURCESDIRECTORY\bootstrap.ps1 " + $bootstrapOptions
+          Write-Host "bootstrapExpression: $bootstrapExpression"
+          Invoke-Expression $bootstrapExpression
+
+          if ($LastExitCode -ne 0) {
+            Write-Host "Bootstrap failed."
+            exit $LastExitCode
+          }
+
+          cmake --build $env:BUILD_BUILDDIRECTORY --config Release -j $env:NUMBER_OF_PROCESSORS
+
+          if ($LastExitCode -ne 0) {
+            Write-Host "Build failed. CMake exit status: " $LastExitCocde
+            exit $LastExitCode
+          }
+
+          cmake --build $env:BUILD_BUILDDIRECTORY  -j $env:NUMBER_OF_PROCESSORS --target install-tiledb --config Release 2>&1
+
+          if ($LastExitCode -ne 0) {
+            Write-Host "Installation failed."
+            $host.SetShouldExit($LastExitCode)
+          }
+        shell: powershell
+
+      - name: 'Test'
+        shell: powershell
+        run: |
+          write-host "begin run: 'Test'"
+
+          $env:MINIO_ACCESS_KEY = "minio"
+          $env:MINIO_SECRET_KEY = "miniosecretkey"
+          $env:AWS_ACCESS_KEY_ID = "minio"
+          $env:AWS_SECRET_ACCESS_KEY = "miniosecretkey"
+          
+          Write-Host "GITHUB_WORKSPACE is '$env:GITHUB_WORKSPACE'"
+          Write-Host "BUILD_BUILDDIRECTORY is '$env:BUILD_BUILDDIRECTORY'"
+          Write-Host "BUILD_SOURCESDIRECTORY is '$env:BUILD_SOURCESDIRECTORY'"
+          
+          # Clone backwards compatibility test arrays
+          if ($env:BACKWARDS_COMPATIBILITY_ARRAYS -eq "ON") {
+            git clone https://github.com/TileDB-Inc/TileDB-Unit-Test-Arrays.git --branch 2.2.3 $env:BUILD_SOURCESDIRECTORY/test/inputs/arrays/read_compatibility_test
+          }
+
+          if ($env:TILEDB_S3 -eq "ON") {
+            # update CMake to disable S3 for the test configuration, see minio note above
+            cmake -B $env:BUILD_BUILDDIRECTORY\tiledb -DTILEDB_S3=0 $env:BUILD_SOURCESDIRECTORY
+          }
+
+          # CMake exits with non-0 status if there are any warnings during the build, so
+          cmake --build $env:BUILD_BUILDDIRECTORY\tiledb  -j $env:NUMBER_OF_PROCESSORS --target tiledb_unit --config Release -- /verbosity:minimal
+
+          if ($env:TILEDB_AZURE -eq "ON") {
+            $useTileDBInstallNodeJS = $False #$True
+            if($useTileDBInstallNodeJS) {

```

Comment:
(if we're going to keep this option - for flexibility I guess? - please use something like `$env:TILEDB_USE_CUSTOM_NODEJS` so it can be set at the matrix level; we might want to test it in a nightly build in the future)

Reply:
done